rye/src/cli/add.rs

```diff
@@ -470,21 +470,8 @@ fn resolve_requirements_with_uv(
         }
         let sources = ExpandedSources::from_sources(&pyproject_toml.sources()?)?;
         sources.add_as_pip_args(&mut cmd);
-        let mut child = cmd
-            .stdin(Stdio::piped())
-            .stdout(Stdio::piped())
-            .stderr(Stdio::piped())
-            .spawn()?;
-        let child_stdin = child.stdin.as_mut().unwrap();
-        writeln!(child_stdin, "{}", req)?;
```

Comment:
If I'm following, the idea was to spawn the child process to write requirements directly to through stdin. So we're going through `uv` compile first before resolving specifiers? Interesting.

Reply:
Hmm. Maybe that's not the entire story. I'm definitely missing something, but I'll come back to this.